packages/mdc-image-list/mdc-image-list.scss

```diff
@@ -0,0 +1,85 @@
+// Copyright 2018 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+@import "./mixins";
+@import "./variables";
+@import "@material/theme/mixins";
+@import "@material/typography/mixins";
+
+.mdc-image-list {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 0 auto; // UA override
+  padding: 0; // UA override
+}
+
+.mdc-image-list__item,
+.mdc-image-list__image-aspect-container {
+  position: relative; // Supports protected text label for item, image for image-aspect-container
+  box-sizing: border-box;
+}
+
+.mdc-image-list__item {
+  list-style-type: none;
+}
+
+.mdc-image-list__image {
+  width: 100%;
+}
+
+// Descendant selector allows image-aspect-container to be optional in DOM structure
+.mdc-image-list__image-aspect-container .mdc-image-list__image {
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  height: 100%;
+  // Background styles to support div instead of img
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+}
+
+@at-root {
+  @include mdc-image-list-aspect(1);
```

Comment:
Why is this under `@at-root` instead of under `.mdc-image-list`?

Reply:
Because it already creates a 1-class selector and there's no reason to make it a 2-class selector. Similar to instances where we moved mixin invocations in other packages' base styles to `@at-root` to decrease specificity and take advantage of BEM.